docs/source-2.0/guides/smithy-cli/cli_installation.rst

```diff
@@ -0,0 +1,470 @@
+.. _cli_installation:
+
+.. |release-uri| replace:: https://github.com/awslabs/smithy/releases/download/__smithy_version__
+.. |mac-tar| replace:: smithy-cli-darwin-x86_64.tar.gz
+.. |mac-arm-tar| replace:: smithy-cli-darwin-aarch64.tar.gz
+.. |linux-tar| replace:: smithy-cli-linux-x86_64.tar.gz
+.. |linux-arm-tar| replace:: smithy-cli-linux-aarch64.tar.gz
+.. |windows-tar| replace:: smithy-cli-windows-x64.tar.gz
+.. |verification-note| replace::
+        Please refer to :ref:`cli_verification` for instructions
+        on how to verify downloaded artifacts.
+
+.. |install-note| replace::
+        If you've already installed the CLI and want to update to the
+        latest version, you can use the ``--update`` flag.
+        You may also choose where to install the CLI - for more information,
+        run the installer using the ``--help`` flag.
+.. |sudo-note| replace::
+        The following command typically requires using ``sudo``
+        to install the CLI in the default location (``/usr/local/smithy``).
+        Alternatively, you may set your own install path, which should
```

Comment:
I might note here that this is done with the `-i` flag on the install script.

Reply:
There's note about that immediately following this